hugegraph-server/hugegraph-example/src/main/java/org/apache/hugegraph/example/PerfExample1.java

```diff
@@ -34,6 +36,8 @@
  */
 public class PerfExample1 extends PerfExampleBase {
 
+    private static final Logger LOG = Log.logger(PerfExample1.class);
```

Comment:
The logger declaration in PerfExample1 and PerfExample2 is added but never used. Consider removing these unused imports and logger declarations to avoid dead code.